components/RecipeListItem.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableHighlight,
+  Image
+} from 'react-native';
+
+export default class RecipeListItem extends Component {
+  constructor(props) {
+    super(props);
+    console.log(this.props.recipe);
+    console.log(this.props.recipe.name);
+    console.log(this.props.recipe.text);
+  }
+
+  render() {
+    return (
```

Comment:
adding `const { onRecipeDelete, recipe } = this.props;` before the return would allow you to reference the props easier like `recipe.name` or `onRecipeDelete`